src/components/common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/StakingWidget/StakingInput/StakingControls/ObjectButton.tsx

```diff
@@ -1,31 +1,33 @@
 import React from 'react';
 
+import RaiseObjectionDialog from '~common/Dialogs/RaiseObjectionDialog';
```

Comment:
We could add an export in the `index` file of the `Dialogs` folder for this dialog to shorten this import